setup.py

```diff
@@ -8,10 +8,5 @@
     description="Long term HPSS archiving software for E3SM",
     packages=find_packages(include=["zstash", "zstash.*"]),
     python_requires=">=3.6",
-    install_requires=[
-        "fair-research-login>=0.2.6,<0.3.0",
-        "globus-sdk>=3.0.0,<4.0.0",
-        "six",
-    ],
```

Comment:
@tomvothecoder It looks like the tests don't like this part being removed. Am I supposed to add these requirements somewhere else?

Reply:
The issue is that the `build` step of the `build_workflow.yml` is using a pip environment and the `install_requires` section from `setup.py`.

You need to replace these lines https://github.com/E3SM-Project/zstash/blob/c0aadc7d7d11b49d3fd60fdc42340b79ee910097/.github/workflows/build_workflow.yml#L38-L41

with the step to cache and create the conda environment ([zppy example](https://github.com/E3SM-Project/zppy/blob/2304054174c723ecd9fb7a94a6a06759d362eee1/.github/workflows/build_workflow.yml#L54-L75)):

```
      - name: Cache Conda
        uses: actions/cache@v3
        env:
          CACHE_NUMBER: 0
        with:
          path: ~/conda_pkgs_dir
          key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{
            hashFiles('conda/dev.yml') }}


      - name: Build Conda Environment
        uses: conda-incubator/setup-miniconda@v2
        with:
          activate-environment: zstash_dev
      miniforge-variant: Mambaforge
          miniforge-version: latest
          use-mamba: true
          mamba-version: "*"
          environment-file: conda/dev.yml
          channel-priority: strict
          auto-update-conda: true
          # IMPORTANT: This needs to be set for caching to work properly!
          use-only-tar-bz2: true
```

Reply:
@tomvothecoder Thanks! Now I'm getting `sphinx-multiversion 0.2.4  does not exist (perhaps a typo or a missing channel).` on the tests though...

Reply:
And everything seems to match up with what was done for `zppy`, which did work.

Reply:
@tomvothecoder Ah, actually `zppy` did not work. I missed a failing GitHub action. Fixing a syntax error in https://github.com/E3SM-Project/zppy/pull/434, I end up with the same error as `zstash` described here.

Reply:
Yes, my latest commit reverting that change makes the GitHub Actions check pass. I may keep it like that to get this merged and address this issue later in the release testing process.

Reply:
The checks appeared to pass for https://github.com/E3SM-Project/zppy/pull/434, but after merging the publish-docs Action failed (https://github.com/E3SM-Project/zppy/actions/runs/5284999456/jobs/9563041218: line 1: mamba: command not found). Because of this, I think it's best to hold off on merging this until the `sphinx-multiversion` error is resolved.